scripts/check-docforge.sh

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bash
```

Comment:
We use an alpine based nodejs image for the check step. There is no bash available. Is it possible to change this to 
```
#!/usr/bin/env sh
```
or do you need bash?

Reply:
I thon no, seems working with sh also